tools/mavlogdump.py

```diff
@@ -379,6 +379,9 @@ def match_type(mtype, patterns):
     elif args.verbose and istlog:
         mavutil.dump_message_verbose(sys.stdout, m)
         print("")
+    elif args.verbose and hasattr(m,"dump_verbose"):
+        m.dump_verbose(sys.stdout)
```

Comment:
I wonder if we should go via `dump_message_verbose` here, even 'though we do have an object (that we control) to call methods on.